tox.ini

```diff
@@ -63,13 +63,6 @@ commands =
 [flake8]
 max-line-length = 2048
 show-source = True
-exclude =
-    env37
-    env38
-    .git
-    .tox
```

Comment:
When I run `tox -e lint` locally, if `.tox` is not excluded, tox appears to hang because it takes so long.  Although it might eventually get over it, it seems like excluding `.tox` is necessary.

Reply:
I'll look into this.